pilot/pkg/bootstrap/server.go

```diff
@@ -28,6 +28,7 @@ import (
 
 	"code.cloudfoundry.org/copilot"
 	"github.com/davecgh/go-spew/spew"
+	"github.com/emicklei/go-restful"
```

Comment:
Oh no - can you just use regular handlers ? We no longer have 'restful' needs.

Reply:
I'll get rid of it once I get tests passing. I seem to have broken a bunch of things, and not sure why.